<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKOpenGLWidget" name="qtvtkWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>671</width>
      <height>431</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>400</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>70</y>

      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AxisButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>190</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Axis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FunctionButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>230</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Function</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadmodelButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>110</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Model</string>
    </property>
   </widget>
   <widget class="QSlider" name="ShrinkFilter">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>610</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderB">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>550</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderG">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>523</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditG">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>520</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditR">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>490</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditB">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>550</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <widget class="QPushButton" name="ClipFilterButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clip</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadmodelButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Model</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BGColourButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Background Colour</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AxisButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Axis</string>
     </property>
    </widget>
    <widget class="QPushButton" name="FunctionButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Function</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>410</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>211</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Statistics</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>621</width>
      <height>111</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEditShrink">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Shrink</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditShadow">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="ShrinkFilter">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Shadow</string>
     </property>
    </widget>
    <widget class="QSlider" name="SliderShadow">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="ListView">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>(Choose Perspective)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>X-Axis</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Y-Axis</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Z-Axis</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>90ยบ Azimuth</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>90ยบ Elevation</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ShrinkButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>270</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Shrink</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>994</width>

     <height>26</height>

    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuColor">
     <property name="title">
      <string>Color</string>
     </property>
     <addaction name="actionModel"/>
     <addaction name="actionBackground"/>
    </widget>
    <addaction name="menuColor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="actionFileOpen">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/fileopen.png</normaloff>:/Icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/filesave.png</normaloff>:/Icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/print.png</normaloff>:/Icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionModel">
   <property name="text">
    <string>Model</string>
   </property>
  </action>
  <action name="actionBackground">
   <property name="text">
    <string>Background</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">qvtkopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
